run_celery.py

```diff
@@ -15,7 +15,8 @@
 application = Flask("celery")
 create_app(application)
 
-xray_recorder.configure(service='celery')
-XRayMiddleware(application, xray_recorder)
+if application.config["AWS_XRAY_ENABLED"] == "true":
+    xray_recorder.configure(service='celery')
+    XRayMiddleware(application, xray_recorder)
```

Comment:
I wonder how that `XRayMiddleware` object works, because we do not make use of it afterward. The xray_celery_handlers do get a hold of the xray_recorder via an import and I guess this might be a singleton? but the XRayMiddleware is just left alone afterward. 🤔